Modules/Server/Sources/PocketCastsServer/Public/Refresh/RefreshManager.swift

```diff
@@ -78,6 +82,40 @@ public class RefreshManager {
             }
         }
     }
+    #endif
+
+    #if os(watchOS)
+    // Watch OS 10 has a very very very weird issue that happened after using Xcode 16:
+    // if the refresh call uses a [weak self] the system kills the app.
+    // I'm not entirely sure why that happens but removing the weak part fixes the issue.
+    // We can safely remove this method once we remove support to watchOS 10.
```

Comment:
Please don't do that for a while. The iPhone app will happily update itself even if the associated watchOS app becomes incompatible, and I haven't found a reasonable way to rollback an app update (I've looked for it over the last 3 months).

Reply:
@DanReyLop we have no intention to remove support for watchOS 10, don't worry :)